docs/tutorials/stateful-application/zookeeper.md

```diff
@@ -1025,22 +980,24 @@ zk-0      0/1       Running   0         51s
 zk-0      1/1       Running   0         1m
 ```
 
-Keep watching the StatefulSet's Pods in the first terminal and drain the node on which 
+Keep watching the `StatefulSet`'s Pods in the first terminal and drain the node on which
 `zk-1` is scheduled.
 
-```shell{% raw %}
+```shell
 kubectl drain $(kubectl get pod zk-1 --template {{.spec.nodeName}}) --ignore-daemonsets --force --delete-local-data "kubernetes-minion-group-ixsl" cordoned
+
 WARNING: Deleting pods not managed by ReplicationController, ReplicaSet, Job, or DaemonSet: fluentd-cloud-logging-kubernetes-minion-group-ixsl, kube-proxy-kubernetes-minion-group-ixsl; Ignoring DaemonSet-managed pods: node-problem-detector-v0.1-voc74
 pod "zk-1" deleted
 node "kubernetes-minion-group-ixsl" drained
-{% endraw %}```
+```
 
-The `zk-1` Pod can not be scheduled. As the `zk` StatefulSet contains a 
-PodAntiAffinity rule preventing co-location of the Pods, and  as only 
+You cannot schedule the `zk-1` Pod. As the `zk` `StatefulSet` contains a
```

Comment:
Kubernetes cannot schedule because ...

Reply:
or revert to passive voice (but keep s/can not/cannot)